WordPress/Classes/ViewRelated/Tags/EditTagView.swift

```diff
@@ -0,0 +1,215 @@
+import SwiftUI
+import WordPressUI
+import WordPressKit
+import WordPressData
+import SVProgressHUD
+
+struct EditTagView: View {
+    @Environment(\.dismiss) private var dismiss
+    @StateObject private var viewModel: EditTagViewModel
+
+    init(tag: RemotePostTag?, tagsService: TagsService) {
+        self._viewModel = StateObject(wrappedValue: EditTagViewModel(tag: tag, tagsService: tagsService))
+    }
+
+    var body: some View {
+        Form {
+            Section(Strings.tagSectionHeader) {
```

Comment:
(nit) I find it easier to read the view code when the `body` only contains top-level screen-related stuff like `navigationTitle`, `toolbar` (if it is a screen), and the rest of it is in `@ViewBuilder var content: some View`:

```swift
 Form {
    content             
}
.toolbar { ... }
```

This screen is simple and short enough though.